Code/ForceField/ANI/ANIForceField_catch.cpp

```diff
@@ -0,0 +1,424 @@
+    auto numAtoms = mol->getNumAtoms();
+    auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(*mol, confId);
+    auto speciesVec = RDKit::Descriptors::ANI::GenerateSpeciesVector(*mol);
+    ForceFields::ForceField field;
+    auto &ps = field.positions();
+    double *pos;
+    pos = new double[numAtoms * 3];
+    for (unsigned int i = 0; i < mol->getNumAtoms(); i++) {
+      auto atom = conf.getAtomPos(i);
+      ps.push_back(&atom);
+      pos[3 * i] = atom.x;
+      pos[3 * i + 1] = atom.y;
+      pos[3 * i + 2] = atom.z;
+
+      auto ac = new ForceFields::ANI::ANIAtomContrib(
+          &field, speciesVec(i), i, speciesVec, numAtoms, 4, 8, "ANI-1x");
+      field.contribs().push_back(ForceFields::ContribPtr(ac));
+    }
+    field.initialize();
+    CHECK(field.calcEnergy(pos) == (-40.0517_a).margin(0.05));
+  }
+  SECTION("NH3") {
+    std::string molFile = dirPath + "NH3.mol";
+
+    RDKit::ROMOL_SPTR mol(RDKit::MolFileToMol(molFile, true, false));
+    int confId = -1;
+    auto conf = mol->getConformer(confId);
+    auto numAtoms = mol->getNumAtoms();
+    auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(*mol, confId);
+    auto speciesVec = RDKit::Descriptors::ANI::GenerateSpeciesVector(*mol);
+    ForceFields::ForceField field;
+    auto &ps = field.positions();
+    double *pos;
+    pos = new double[numAtoms * 3];
+    for (unsigned int i = 0; i < mol->getNumAtoms(); i++) {
+      auto atom = conf.getAtomPos(i);
+      ps.push_back(&atom);
+      pos[3 * i] = atom.x;
+      pos[3 * i + 1] = atom.y;
+      pos[3 * i + 2] = atom.z;
+
+      auto ac = new ForceFields::ANI::ANIAtomContrib(
+          &field, speciesVec(i), i, speciesVec, numAtoms, 4, 8, "ANI-1x");
+      field.contribs().push_back(ForceFields::ContribPtr(ac));
+    }
+    field.initialize();
+    CHECK(field.calcEnergy(pos) == (-56.1592_a).margin(0.05));
+  }
+  SECTION("Ethanol") {
+    std::string molFile = dirPath + "ethanol.sdf";
+
+    RDKit::ROMOL_SPTR mol(RDKit::MolFileToMol(molFile, true, false));
+    int confId = -1;
+    auto conf = mol->getConformer(confId);
+    auto numAtoms = mol->getNumAtoms();
+    auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(*mol, confId);
+    auto speciesVec = RDKit::Descriptors::ANI::GenerateSpeciesVector(*mol);
+    ForceFields::ForceField field;
+    auto &ps = field.positions();
+    double *pos;
+    pos = new double[numAtoms * 3];
+    for (unsigned int i = 0; i < mol->getNumAtoms(); i++) {
+      auto atom = conf.getAtomPos(i);
+      ps.push_back(&atom);
+      pos[3 * i] = atom.x;
+      pos[3 * i + 1] = atom.y;
+      pos[3 * i + 2] = atom.z;
+
+      auto ac = new ForceFields::ANI::ANIAtomContrib(
+          &field, speciesVec(i), i, speciesVec, numAtoms, 4, 8, "ANI-1x");
+      field.contribs().push_back(ForceFields::ContribPtr(ac));
+    }
+    field.initialize();
+    CHECK(field.calcEnergy(pos) == (-154.9892_a).margin(0.05));
+  }
+}
+TEST_CASE("QM7 Test Cases") {
```

Comment:
This is a test case which runs an energy calculation.  

![image](https://user-images.githubusercontent.com/1225605/87790482-9c8ff980-c80e-11ea-9c13-abb316c990ee.png)

These are the runtimes we are seeing.